docs/reference/errors-and-warnings/NU1011.md

```diff
@@ -15,7 +15,18 @@ f1_keywords:
 
 ### Issue
 
-PackageVersion items cannot contain floating versions.
+`<PackageVersion />` items cannot contain floating versions.  NuGet's central package management (CPM) is considered an enterprise-level feature which provides easier version
+management at scale as well as deterministic and secure restores.  The use of floating versions introduces the possibility for a bad package to be introduced into your build
+after it has been pushed to a feed.  This can lead to a situation where you made no changes in your repository but suddenly something is broken and there is no way for you to
+get back into a good state without removing the floating version or pushing a newer version of the package which is fixed.  Using non-floating versions means that every upgrade
+to a package is backed by a commit in your repository making it easy to determine what change caused the break and to revert a commit to get back into a good state.
+
+Features of central package management like [Transitive Pinning](../../consume-packages/Central-Package-Management.md#transitive-pinning) is designed to allow you to explicitly override the transitive versions in your graph for more control. Using a floating version as an override would lead to very undeterministic restores.  [NuGet Lock Files](../../consume-packages/package-references-in-project-files.md#enabling-the-lock-file) are another enterprise-level feature that does not work well with floating versions since the lock files will be constantly need to be regenerated.

```

Comment:
This isn't fully correct as we discussed in the review meeting. I'll write a proposal.

Reply:
```suggestion
The [Transitive Pinning](../../consume-packages/Central-Package-Management.md#transitive-pinning) feature is designed to allow you to explicitly override the transitive versions in your graph for more control. Using a floating version as an override could make restores of different projects end up with different versions for the package that is supposed to be pinned, thus going against the promise of using the central version.
```

Sorry for the delay. How's this? Thinking about it more, maybe we don't need to mention lock files at all.

Reply:
Looks good, thanks!